src/widgets.ts

```diff
@@ -28,12 +28,14 @@ export abstract class Widget<
   protected container: Element
   protected style: Partial<CSSStyleDeclaration>
   protected isUnmounting: boolean
+  protected ssoUrl: string
```

Comment:
Nit: let's turn this into `protected ssoUrl?: string` since it won't always have a valid value. This also let's the default null value represent a lack of a URL, and a string represents the current SSO URL.